p34/src/lib.rs

```diff
@@ -0,0 +1,130 @@
+use std::ops;
+
+#[derive(Debug)]
```

Comment:
You can derive `Default` and `Hash` here. For printing it's also worth to implement the hex formatting traits.

Reply:
I couldn't implement it automatically

```rust
impl Default for BigUint4096 {
    fn default() -> Self {
        Self { values: [0; 64] }
    }
}
```

Reply:
solved in bd3e8907a4fb2a6a14aafa21b90b6aaef8b3d17b

Reply:
- [ ] hex formatting traits.